docs/en/05_Contributing/02_Request_for_comment.md

```diff
@@ -0,0 +1,52 @@
+# Request for comment (RFC)
+
+## Why RFCs?
+This is part of the SilverStripe core decision-making process and addresses the gap between the idea phase and the pull request submission (and merge).
+
+The rationale behind this process is to:
+ * Encourage visibility on decision making
+ * Clarity on what the proposal is, its rationale and impact
+ * Reduce unnecessary work when coding large features is done without community and core-committers buy-in
+ * Improved likelihood of an optimal solution being merged
+
+The important thing to understand about the RFCs is that these are NOT a way to request features. Rather, they are a way to bring clarity so people understand what change is being proposed.
+
+## When to write an RFC?
+
+We intend RFCs to be the primary mechanisms for proposing major new features, for collecting community input on an issue, and for documenting the design decisions that have gone into SilverStripe. The RFC author is responsible for building consensus within the community and documenting dissenting opinions.
+
+Before writing the actual summary RFC document, the idea should have already had a wide range of discussion in various community communication channels. Once discussions reach a point where you think most of the difficulties have been worked through then create the RFC using the template provided.
+
+The benefits of writing an RFC for non-trivial feature proposals are:
+ * Obtaining a preliminary approval from core-committers on an architecture before code is completed, to mitigate the risk of a non-merge after a PR is submitted
+ * Community becomes aware of incoming changes prior to the implementation
+ * RFC can be used as a basis for documentation of the feature
+	
+## How to write an RFC?
+### Template
```

Comment:
This might be a little clearer:
```
The following structure can act as a template for starting your RFC.
 * **Title** - short descriptive title which includes reference number e.g. *RFC-4: Client-side dependency management*.
 * **Metadata** - standardised header containing at least the Author(s), Status, Version and Last Updated fields.
 * **Purpose and outcome** - briefly describe the problem and solution your RFC addresses.
 * **Motivation** - describe the problem in more detail.
 * **Proposal** - describe the solution in more detail, including any questions or considerations that will need to be addressed during development.
 * **Alternatives** - describer any other approaches that were considered during the community discussion phase and why they were not chosen.
 * **Impact** - How will this change potentially impact SilverStripe core? List any positive and negative effects and upgrade considerations.
```